### PR TITLE
Help mode

### PR DESCRIPTION
**Pull request by maxsandova**
Added an alternative to "RET"

### CODE REVIEW

**Comment by jojojames** [comment]
Can you squash these commits into one?

You can squash and then force push your branch and it'll appear as one commit when all is said and done.

File: evil-collection-help.el [hunk 1]
```diff
@@ -51,6 +51,8 @@
     ;; "\C-j" 'help-go-forward
     ;; "\C-k" 'help-go-back
 
+    "go" 'push-button ;; this should open in same window
+    "gO" 'push-button ;; this should open in another window
```
Comment (jojojames) [inline evil-collection-help.el]: Are these comments true? These both trigger 'push-button'.

Comment (maxsandova) [reply evil-collection-help.el]: thats why I put should.  Neither do what they should `push-button` sometimes opens in the same window sometimes in other-window context-wise.

Comment (jojojames) [reply evil-collection-help.el]: OK, it might be better to clarify the comments here.

Comment (maxsandova) [reply evil-collection-help.el]: Ok, do you have a suggestion on implementation? I don't see the point on writing two functions here, I would just bind both as in the commit.

Comment (jojojames) [reply evil-collection-help.el]: You can just update the comments here to reflect that they 'should' open in same/another window *but* don't in all cases.

Comment (maxsandova) [reply evil-collection-help.el]: Ok, that I will do.

**Comment by jojojames** [comment]
Thanks.

**Comment by maxsandova** [comment]
Thanks to you, without realizing it, you teached me git, cask, code etiquette among other things.

**Comment by jojojames** [comment]
@maxsandova Thanks, it was all you. :)

Last nit for your edification, you might want to take a look at https://github.com/erlang/otp/wiki/writing-good-commit-messages

It's not the *only* way to do things, but it's *one* nice way to do it.

**Pull request merged**